src/huggingface_hub/hf_api.py

```diff
@@ -10453,6 +10453,525 @@ def run_uv_job(
             token=token,
         )
 
+    def schedule_job(
+        self,
+        *,
+        image: str,
+        command: List[str],
+        schedule: str,
+        suspend: bool = False,
+        concurrency: bool = False,
```

Comment:
```suggestion
        suspend: Optional[bool] = None,
        concurrency: Optional[bool] = None,
```

Better to default to `None` and pass a value in the body only if non-none value is passed. This way we let the server handle the default value